src/config/resolve-config.js

```diff
@@ -165,18 +165,30 @@ function mergeOverrides(configResult, filePath) {
 }
 
 // Based on eslint: https://github.com/eslint/eslint/blob/master/lib/config/config-ops.js
-function pathMatchesGlobs(filePath, patterns, excludedPatterns = []) {
+function pathMatchesGlobs(filePath, patterns, excludedPatterns) {
   const patternList = Array.isArray(patterns) ? patterns : [patterns];
-  const excludedPatternList = Array.isArray(excludedPatterns)
-    ? excludedPatterns
-    : [excludedPatterns];
-  const opts = { matchBase: true, dot: true };
-
+  // micromatch always matches against basename when the option is enabled
+  // use only patterns without slashes with it to match minimatch behavior
+  const withSlashes = [];
+  const withoutSlashes = [];
+  for (const pattern of patternList) {
+    if (pattern.includes("/")) {
+      withSlashes.push(pattern);
+    } else {
+      withoutSlashes.push(pattern);
+    }
+  }
```

Comment:
Will it increase bundle size if we use `lodash.partition()`? It's already used in other place.

Reply:
I would honestly try to move away from lodash. It handles a lot of unused cases with tone of code. And readability may be affected as well. Caz I have not idea how partition works and will need to use docs. While here everybody aware of for/of and arrays. I just don't see the value in using lodash with modern js.

Reply:
I agree with you, I suggest this only because we already used `lodash.partition()`.